modules/auxiliary/scanner/http/s40_traversal.rb

```diff
@@ -55,10 +55,8 @@ def run_host(ip)
     })
 
     if not res
-      vprint_error("Server timed out")
-    elsif res and res.body =~ /Error 404 requested page cannot be found/
-      vprint_error("Either the file doesn't exist, or you don't have the permission to get it")
-    else
+      print_error("Server timed out")
+    elsif res.code == 200
```

Comment:
Do we know the software follows this?  Sometimes codes will be response code 200 with error 404 in the body.

Reply:
i didn't know that. so i resolve it.

Reply:
i don't know about it. then how can i change my code in that case?

Reply:
> i don't know about it. then how can i change my code in that case?

You would need to verify on a live vulnerable installation.